Adding changes of commit 23ef8f6

AnkithB2020 opened the pull request:
Commit 23ef8f6 includes changes related to replacement of
os.mkfs() with os.fsformat()

Link explaining the replacement details:
https://forum.pycom.io/topic/3351/new-development-firmware-release-v1-19-0-b1/2

Issue: https://forum.pycom.io/topic/5606/what-happened-to-pin_deepsleep_wakeup/9

AnkithB2020 committed:
Adding changes of commit 23ef8f6

Commit 23ef8f6 includes changes related to replacement of
os.mkfs() with os.fsformat()

Link explaining the replacement details:
https://forum.pycom.io/topic/3351/new-development-firmware-release-v1-19-0-b1/2

Issue: https://forum.pycom.io/topic/5606/what-happened-to-pin_deepsleep_wakeup/9

lilycey commented on content/tutorials/lte/firmware.md:
```diff
@@ -37,7 +37,7 @@ Please start with the following steps:
 
 
-The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to: 
+The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to:
```
1st comment: The modem firmware files are password protected. In order to download them, head to  forum.pycom.io and become a member (if you aren't already) and click on:

AnkithB2020 replied:
Comment incorporated.

lilycey commented on content/tutorials/lte/firmware.md:
```diff
@@ -37,7 +37,7 @@ Please start with the following steps:
 
-The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to: 
+The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to:
 Announcements & News --&gt;  Announcements only for members --&gt; Firmware Files for Sequans LTE modem now are secured, or clicking [Here](https://forum.pycom.io/topic/4020/firmware-files-for-sequans-lte-modem-now-are-secured).
```
2nd comment: Announcements & News --&gt;  Announcements for members only --&gt;  the Firmware Files for the Sequans LTE modem are now secured or click [here](https://forum.pycom.io/topic/4020/firmware-files-for-sequans-lte-modem-now-are-secured).

AnkithB2020 replied:
Comment incorporated.

AnkithB2020 replied:
Comment incorporated.

lilycey commented on content/tutorials/lte/firmware.md:
```diff
@@ -37,7 +37,7 @@ Please start with the following steps:
+The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to:
 Announcements & News --&gt;  Announcements only for members --&gt; Firmware Files for Sequans LTE modem now are secured, or clicking [Here](https://forum.pycom.io/topic/4020/firmware-files-for-sequans-lte-modem-now-are-secured).
 
 You can find the different versions of firmwares available [here](https://software.pycom.io/downloads/sequans2.html).
```
3rd comment: You can find the different available firmware versions [here](https://software.pycom.io/downloads/sequans2.html).

lilycey commented on content/tutorials/lte/firmware.md:
````diff
@@ -57,7 +57,7 @@ import sqnsupgrade
 sqnsupgrade.run('upgdiff_38638-to-39529.dup')
 ```
 If you are updating the Sequans firmware on your module for the first time, please use instead the file upgdiff_33080-to-39529.dup (1.2M) from the same archive.
````
4th comment: If you are updating the Sequans firmware on your module for the first time, please use this file instead  upgdiff_33080-to-39529.dup (1.2M), which is from the same archive.

lilycey commented on content/tutorials/lte/firmware.md:
````diff
@@ -57,7 +57,7 @@ import sqnsupgrade
 ```
 If you are updating the Sequans firmware on your module for the first time, please use instead the file upgdiff_33080-to-39529.dup (1.2M) from the same archive.
-Similar upgrade packages are available for the NB-IoT firmwares. 
+Similar upgrade packages are available for the NB-IoT firmwares.
````
5th: Similar upgrade packages are available for the NB-IoT firmware.

lilycey commented on content/tutorials/lte/firmware.md:
```diff
@@ -267,5 +270,3 @@ The latest version of the `sqnsupgrade` class has a few additional features that
 
 #### sqnsupgrade.run(load_fff=True)
     New optional command line option load_fff for the sqnsupgrade.run() command. This is designed to be an internal flag. And should only applied when advised by pycom support.
```
6th: There is an optional command line load_fff for the sqnsupgrade.run() command. This is designed to be an internal flag. IMPORTANT: This should only be used when advised by Pycom Support.

lilycey reviewed (commented):
Updated to improve readability

jirkadev commented:
@AnkithB2020 I think you need to open also a second PR

AnkithB2020 committed:
Update firmware.md: Updated for better readability

Incorporated review comments to update firmware.md for better readability.

jirkadev commented on content/tutorials/lte/firmware.md:
```diff
@@ -37,10 +37,10 @@ Please start with the following steps:
+Announcements & News --> Announcements for members only --> the Firmware Files for the Sequans LTE modem are now secured or click here.
 
-You can find the different versions of firmwares available [here](https://software.pycom.io/downloads/sequans2.html).
+You can find the different available firmware versions here.
```
We need to keep the URL for that markdown link there 

`[here](https://software.pycom.io/downloads/sequans2.html)`

AnkithB2020 replied:
Ok, sure, I have kept the URL for the markdown link now.

AnkithB2020 replied:
Also, .DS_Store file has been removed.

jirkadev replied:
Actually, the Here links were not fixed consider how it looks in your PR and in production docs.

Please install and use hugo on your pc and `hugo serve` to preview your changes locally

<img width="2032" alt="Screen Shot 2020-01-29 at 14 16 33" src="https://user-images.githubusercontent.com/2588479/73360090-7ac3aa00-42a2-11ea-9ebe-5a27a8346816.png">
<img width="2032" alt="Screen Shot 2020-01-29 at 14 17 01" src="https://user-images.githubusercontent.com/2588479/73360091-7b5c4080-42a2-11ea-9805-2e0e7a3c0657.png">

jirkadev reviewed (commented):
Thank you for your changes. Please remove .DS_Store files from your commit.

AnkithB2020 committed:
Update firmware.md: Put back the URL for the markdown link

Put back the URL: https://software.pycom.io/downloads/sequans2.html) for the markdown link.

jirkadev commented:
I still see DS_store files here in this PR. If you go to "files changed" you would see them too

Xykon committed:
Remove .DS_Store files

Xykon committed:
Update firmware.md

Fix link to firmware downloads
